Library/Formula/rcssserver.rb

```diff
@@ -67,7 +67,31 @@ def install
 
     resources.each do |r|
       r.stage do
-        system "./bootstrap" if build.head?
+        Patch.create(:p1, <<EOD).apply
```

Comment:
I created a Gist file.
URL is a follow.
https://gist.githubusercontent.com/tknrkb/2473eb101a57a457b51d/raw/3bad9c9e44b943a83508eda1f0a26db5402601c8/rcssmonitor_logplayer_qt_m4.patch

Is it OK?

But, I don't know how to write formula to apply patch for resources.
Please show me example codes or documents of that.